Package.swift

```diff
@@ -1,19 +1,28 @@
-//    Copyright 2016 Swift Studies
-//
-//    Licensed under the Apache License, Version 2.0 (the "License");
-//    you may not use this file except in compliance with the License.
-//    You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-//    Unless required by applicable law or agreed to in writing, software
-//    distributed under the License is distributed on an "AS IS" BASIS,
-//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//    See the License for the specific language governing permissions and
-//    limitations under the License.
+// swift-tools-version:5.2
```

Comment:
This line and the .gitignore changes seem to be the only things actually relevant to using swift 5.2 and Xcode (nothing looks specific to Xcode 11).

Reply:
OK, I rollback `.gitignore`.

I tried to built this repo as `swift build`, however I got an error as follows.
```
error: package at '/Users/akira/Documents/github/iOS/Duration' is using Swift tools version 3.1.0 which is no longer supported; consider using '// swift-tools-version:5.2' to specify the current tools version
```
So I regenerate Swift Package using Xcode 11 which includes Swift 5.2.
In Xcode 11.4 in my environment, it is possible to use Swift 4.0 to build this package.
Should I change this comment to `// swift-tools-version:4.0`?
In any case I can build this package by adding such comment to control swift version, i.e. it is not possible to build without the comment.